browser/app/js/web.js

```diff
@@ -59,6 +69,17 @@ class Web {
         return result
       })
   }
+  createTranslatedError(error) {
+    let key = error.key
+    if(error.data){
+      //As is, there can only be one parameter coming with data, and it has the key 'param'
+      //for(var k in error.data){
+      //  return i18next.t(key, {param:error.data[k]})
+      //}
+      return i18next.t(key, {param:error.data['param']})
+    }
+    return i18next.t(key)
+  }
```

Comment:
I generally like this approach. Some thoughts:
* Could have `data` itself contain `key`, which would preserve the existing shape of `JSONError`, if that's desirable.
* If `key` goes into `data` as per above, then we could do `.t(key, parameters)` and have the front-end message parameters conform to what comes from the server.
* If key stays where it is, I think we can get rid of `param` and just make it `data` directly.

Reply:
This makes more sense, and allows for multiple params in the string, if need be.